docs/reference/ingest/apis/put-geoip-database.asciidoc

```diff
@@ -0,0 +1,72 @@
+[[put-geoip-database-api]]
+=== Create or update geoip database configuration API
+++++
+<titleabbrev>Create or update geoip database configuration</titleabbrev>
+++++
+
+Creates or updates an ip geolocation database configuration.
+
+IMPORTANT: The Maxmind `account_id` shown below requires a license key. Because the license key is sensitive information,
```

Comment:
The [secure settings](https://www.elastic.co/guide/en/elasticsearch/reference/current/secure-settings.html) page is a little odd, in that more than half of it is dedicated specifically to reloadable secure settings, and all of those settings are listed there because they are reloadable settings. I'll see if I can find a place to highlight `ngest.geoip.downloader.maxmind.license_key`.

Reply:
Ok disregard this for now then, thank you!